File-Storage-Security/Post-Scan-Action-Automations/aws-slack-notification-automation/ReadMe.md

```diff
@@ -0,0 +1,32 @@
+# Subscribe deployed FSS Storage Stack SNS topics to Slack Notification Plugin.
```

Comment:
Small comment: 
- Keep the file nomeclature as `README.md` just to keep the standard with the rest of the repo.

Reply:
fixed